front/src/modules/ui/dropdown/components/DropdownMenu.tsx

```diff
@@ -83,29 +84,36 @@ export const DropdownMenu = ({
     dropdownHotkeyScope.scope,
     [closeDropdown],
   );
+
   return (
-    <div ref={containerRef}>
-      {clickableComponent && (
-        <div ref={refs.setReference} onClick={toggleDropdown}>
-          {clickableComponent}
-        </div>
-      )}
-      {hotkey && (
-        <HotkeyEffect
-          hotkey={hotkey}
-          onHotkeyTriggered={handleHotkeyTriggered}
+    <DropdownScope dropdownScopeId={dropdownScopeId}>
```

Comment:
Ok, this makes sense. here we are passing dropdownScopeId to the dropdownMenu. We could defend instanciating the <DropdownScope> out of the DropdownMenu. This would allow useDropdown() without passing a dropdown scopeId